control/autoware_joy_controller/launch/autoware_joy_controller.launch.xml

```diff
@@ -24,7 +24,7 @@
     <remap from="output/shift" to="$(var output_shift)"/>
     <remap from="output/turn_signal" to="$(var output_turn_signal)"/>
     <remap from="output/gate_mode" to="$(var output_gate_mode)"/>
-    <remap from="output/emergency" to="$(var output_emergency)"/>
+    <remap from="output/emergency_stop" to="$(var emergency_stop)"/>
```

Comment:
var emergency_stop -> var output_emergency_stop?

Reply:
thank you! 19af154